packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -30,6 +30,7 @@ const Map<String, String> _kManuallyPinnedDependencies = <String, String>{
   'vm_service_client': '0.2.6+2', // Final version before being marked deprecated.
   'video_player': '0.10.6', // 0.10.7 fails a gallery smoke test for toString.
   'flutter_template_images': '1.0.1', // Must always exactly match flutter_tools template.
+  'shelf': '0.7.5'
```

Comment:
do you remember why we pinned this?
(i'm trying to add comments to the file explaining all the pins)